src/main/java/org/fcrepo/spec/testsuite/crud/ExternalBinaryContent.java

```diff
@@ -422,6 +859,15 @@ public void binaryContentRedirectWantDigest(final String uri) {
                 "Fedora servers supporting \"redirect\" external content types MUST correctly respond to the " +
                         "\"Want-Digest\" header.",
                 "https://fedora.info/2018/06/25/spec/#redirect-and-proxy", ps);
+
+        final String location = createExternalBinary(uri, info.getId(), "redirect", null);
+
+        final String checksum = "md5;q=0.3,sha;q=1";
+        final Response wantDigestResponse = doGetUnverified(location, new Header("Want-Digest", checksum));
+
+        final Headers responseHeaders = wantDigestResponse.getHeaders();
+        assertTrue(responseHeaders.getValue(DIGEST).contains("md5") ||
+                responseHeaders.getValue(DIGEST).contains("sha"), "OK");
```

Comment:
The `"OK"` message may be misleading, as the message is only displayed on failure. I would suggest removing the message, or adding a message that says something like, "Expected Want-Digest value not found".

Reply:
This was mainly borrowed from https://github.com/fcrepo/Fedora-API-Test-Suite/blob/master/src/main/java/org/fcrepo/spec/testsuite/crud/HttpGet.java#L228-L254 but i'm realizing now that that was a more complicated case than I need here,  I can probably mimic https://github.com/fcrepo/Fedora-API-Test-Suite/blob/master/src/main/java/org/fcrepo/spec/testsuite/crud/HttpGet.java#L202-L218 instead

Reply:
And I just also realized that you were referring to the message displayed when the assertion fails :) I keep forgetting that with testng the message goes after the condition. Too much time in junit land.